### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by tudorpip**
# Pull Request Template

## Type of change

-   [ ] 🍕 New feature
-   [ ] 🐛 Bug Fix
-   [ ] 🔥 Breaking change
-   [x] 🧑‍💻 Improvement
-   [ ] 📝 Documentation Update

## Description

Improved deployment logging for nuxt and nitro

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [ ] I have updated the documentation;
-   [ ] I have added tests;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Review by andreia-oca — APPROVED** [review]
LGTM

File: src/commands/deploy/nitro/deploy.ts [hunk 1]
```diff
@@ -71,5 +73,8 @@ async function deployFunctions(config: YamlProjectConfiguration, stage?: string)
     ]);
     debugLogger.debug(`Deployed functions: ${JSON.stringify(result.functions)}`);
 
+    log.info(
+        `The function is being deployed at ${colors.cyan(JSON.stringify(result.functions[0].cloudUrl))}. It might take a few moments to be available worldwide.`,
```
Comment (costinsin) [inline src/commands/deploy/nitro/deploy.ts:77]:
```suggestion
        `${colors.cyan(Your backend code was successfully deployed)}
        
The server function endpoint is available at ${colors.cyan(result.functions[0].cloudUrl)}`,
```

File: src/commands/deploy/nuxt/deploy.ts [hunk 2]
```diff
@@ -71,5 +73,9 @@ async function deployFunctions(config: YamlProjectConfiguration, stage?: string)
     ]);
     debugLogger.debug(`Deployed functions: ${JSON.stringify(result.functions)}`);
 
+    log.info(
+        `The function is being deployed at ${colors.cyan(JSON.stringify(result.functions[0].cloudUrl))}. It might take a few moments to be available worldwide.`,
```
Comment (costinsin) [inline src/commands/deploy/nuxt/deploy.ts:77]:
```suggestion
        `${colors.cyan(Your Nuxt code was successfully deployed)}
        
Your Nuxt app is available at ${colors.cyan(result.functions[0].cloudUrl)}`,
```

**Review by costinsin — CHANGES_REQUESTED** [review]
The message is copied from Next.js, but Nitro and Nuxt are different. The `It might take a few moments to be available worldwide.` message does not apply to these two.

**Pull request merged**